src/sdk/services/communication/message.ts

```diff
@@ -255,3 +255,59 @@ export class LinkAuthRequestResponseMessage extends Message<LinkAuthRequestRespo
         return new LinkAuthRequestResponseMessage(vc);
     }
 }
+
+export interface VerificationMessagePayload {
+    veriffId: string;
+    vc: {
+        kyc: VerifiableCredentialWithType<{ kyc: string }>;
```

Comment:
`<{ kyc: string }>` this is not the type, as there are actually a bunch of properties within this (VeriffWebhookPayload)